src/style.css

```diff
@@ -0,0 +1,132 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 600;
+  font-style: normal;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  margin: 0;
+}
+
+input {
+  font-family: "Avenir", sans-serif;
+}
+
+.form__big {
+  margin: 20px 8px;
+}
+
+.form__small-input,
+.form__big-input {
+  width: 100%;
+}
+
+.form__big-input {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61,78,97,0.1);
+  border-radius: 4px;
+  outline: none;
+  font-weight: 300;
+}
+
+.form__big-input:hover {
+  box-shadow: 0 3px 8px rgba(61,78,97,0.2);
+}
+
+.form__big-input:focus {
+  background: linear-gradient(180deg,#fff,#f6f6f7);
+  border-radius: 3px;
+  font-weight: 600;
+}
+
+.form__big-input::placeholder {
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.form__big-label {
+  position: relative;
+  display: block;
+}
+
+.form__big-label::before {
```

Comment:
Follow the checklist, please. Don't repeat the same styles 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/99403198/ad93c919-6359-43c6-91ab-7fab681242f3)